src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -395,6 +287,89 @@ private IEnumerable<TypeInferenceInfo> InferTypeInConstructorInitializer(Constru
                 return InferTypeInArgument(index, methods, argument, parentInvocationExpressionToTypeInfer: null);
             }
 
+            private IEnumerable<TypeInferenceInfo> InferTypesWorkerFromParent(SyntaxNode parent, SyntaxToken? token = null, SyntaxNode node = null, ExpressionSyntax expression = null)
```

Comment:
Can you make all the parameters non-optional plz.

Reply:
`parent, node, expression, token` for the order please.  thanks.

Reply:
Sure, will make the change.